src/glider/glider.controller.ts

```diff
@@ -15,6 +15,12 @@ export class GliderController {
         return this.gliderFacade.getGliders(req.user, query);
     }
 
+    @UseGuards(JwtAuthGuard)
+    @Get('name/:name')
```

Comment:
Maybe just  @Get(':name') like used at getPlacesByName?

Reply:
That's what I wanted to do first. But then I decided against it because it was no longer possible to make glider/:id. In the rest concept resource / id should always return specific entity